openrtb_ext/imp_ccx.go

```diff
@@ -0,0 +1,5 @@
+package openrtb_ext
+
+type ExtImpCcx struct {
+	PlacementID string `json:"pid"`
```

Comment:
Could you rename the json path to be `json:"placementId"`

Reply:
https://github.com/prebid/prebid.github.io/pull/4065

Reply:
Is that necessary? 
My placement param is pid in static\bidder-params\ccx.json
Do i not understand something?

Reply:
Done